bindings/csharp/IioLib.cs

```diff
@@ -2,7 +2,7 @@
  * libiio - Library for interfacing industrial I/O (IIO) devices
  *
  * Copyright (C) 2020 Analog Devices, Inc.
- * Author: Paul Cristian Iacob <cristian.iacob@analog.com>
+ * Author: Cristian Iacob <cristian.iacob@analog.com>
```

Comment:
actually his name really is Paul Cristian Iacob.

Reply:
Ha :D
All other headers just had "Cristian Iacob", so I thought it was a bad copy-paste.